common/src/main/java/com/wynntils/mc/mixin/GuiMixin.java

```diff
@@ -167,4 +170,33 @@ private void onRenderHeartsPre(
             ci.cancel();
         }
     }
+
+    @Inject(
+            method = "Lnet/minecraft/client/gui/Gui;renderSelectedItemName(Lnet/minecraft/client/gui/GuiGraphics;)V",
+            at = @At("HEAD"),
+            cancellable = true)
+    private void onRenderSelectedItemNamePre(GuiGraphics guiGraphics, CallbackInfo ci) {
+        if (!MixinHelper.onWynncraft()) return;
+
+        RenderEvent.Pre event =
+                new RenderEvent.Pre(guiGraphics, 0, this.minecraft.getWindow(), RenderEvent.ElementType.HELD_ITEM_NAME);
```

Comment:
Help me out. For the other ElementTypes, do we always fire a Pre/Post event, or just the ones we use? If we do, please fix it here. If not, well, I hope it won't cost someone time in the future.

Reply:
Health bar, food bar, tab overlay and crosshair (Do we even do anything with that?) all only have pre. The `render` method is the only one to have pre and post